nonthermal.cc

```diff
@@ -2727,4 +2777,4 @@ void nt_print_stats(const double modelvolume, const double deltat) {
   printout("nt_energy_deposited = %g [eV/s/cm^3]\n", deposition_rate_density_montecarlo);
 }
 
-}  // namespace nonthermal
\ No newline at end of file
+}  // namespace nonthermal
```

Comment:
Check that you have pre-commit installed correctly, which should make sure that all text files end with a single newline.